superset/cli/viz_migrations.py

```diff
@@ -46,37 +47,43 @@ def migrate_viz() -> None:
 
 @migrate_viz.command()
 @with_appcontext
-@optgroup.group(
-    "Grouped options",
-    cls=RequiredMutuallyExclusiveOptionGroup,
-)
-@optgroup.option(
+@click.option(
     "--viz_type",
     "-t",
-    help=f"The viz type to migrate: {', '.join(list(VizType))}",
+    help=f"The viz type to upgrade: {', '.join(list(VizType))}",
+    required=True,
+)
+@click.option(
+    "--chart_id",
+    help="The chart ID to upgrade",
+    type=int,
 )
-def upgrade(viz_type: str) -> None:
+def upgrade(viz_type: str, chart_id: int | None = None) -> None:
```

Comment:
Should this be something which can be specified [multiple times](https://click.palletsprojects.com/en/8.1.x/options/#multiple-options)? Also I wonder if `--chart` would be suffice, i.e., it's somewhat implicit that you would be specifying the chart ID.

Reply:
Actually given the context (migrating a visualisation) maybe `--id` would make more sense and be more in line with `-t` for viz type.

Reply:
I updated the description with:

```
--ids TEXT           A comma separated list of chart IDs to downgrade
```